source/_posts/2023-09-06-release-20239.markdown

```diff
@@ -0,0 +1,485 @@
+  hourly and daily forecast.
+
+([@emontnemery] - [#98022]) ([documentation](/integrations/meteo_france))
+
+[@emontnemery]: https://github.com/emontnemery
+[#98022]: https://github.com/home-assistant/core/pull/98022
+
+{% enddetails %}
+
+{% details "Mill" %}
+
+Mill has moved to a new IoT solution and released a new API. The unique ID from
+the old API is no longer available, so all devices will be shown as new devices.
+Support for the heater fan will be added later.
+
+([@Danielhiversen] - [#97497]) ([documentation](/integrations/mill))
+
+[@Danielhiversen]: https://github.com/Danielhiversen
+[#97497]: https://github.com/home-assistant/core/pull/97497
+
+{% enddetails %}
+
+{% details "MQTT" %}
+
+MQTT device trackers no longer always update the when there is no state change.
+This means that automations that trigger on a state change no longer are
+triggered if there is no change.
+
+([@jbouwh] - [#98838]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#98838]: https://github.com/home-assistant/core/pull/98838
+
+{% enddetails %}
+
+{% details "Nextcloud" %}
+
+The CPU load sensor has been replaced, it is not a list of 3 values anymore,
+but there are now three single sensors for
+CPU load within last 1, 5, and 15 minutes. 
+
+([@mib1185] - [#99141]) ([documentation](/integrations/nextcloud))
+
+[@mib1185]: https://github.com/mib1185
+[#99141]: https://github.com/home-assistant/core/pull/99141
+
+{% enddetails %}
+
+{% details "Plugwise" %}
+
+The `maximum_boiler_temperature` sensor entity has been replaced by a number
+entity, allowing control as well.
+
+([@CoMPaTech] - [#97203]) ([documentation](/integrations/plugwise))
+
+[@CoMPaTech]: https://github.com/CoMPaTech
+[#97203]: https://github.com/home-assistant/core/pull/97203
+
+---
+
+The additional schema state attributes of the climate entity have been removed.
+They have beeen previously replaced by a select entity.
+
+([@bouwew] - [#98153]) ([documentation](/integrations/plugwise))
+
+[@bouwew]: https://github.com/bouwew
+[#98153]: https://github.com/home-assistant/core/pull/98153
+
+{% enddetails %}
+
+{% details "Start.ca" %}
+
+Start.ca `limit`, `usage`, and `used_remaining` sensors will no longer be created
+when there's no bandwidth cap.
+
+([@emontnemery] - [#98525]) ([documentation](/integrations/startca))
+
+[@emontnemery]: https://github.com/emontnemery
+[#98525]: https://github.com/home-assistant/core/pull/98525
+
+{% enddetails %}
+
+{% details "Trafikverket Train" %}
+
+All attributes have been converted into their own sensors.
+The original sensor is now renamed with suffix "Departure time".
+
+Any automation or script depending on these attributes needs to be adjusted
+to use these new sensors.
+
+([@gjohansson-ST] - [#71432]) ([documentation](/integrations/trafikverket_train))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#71432]: https://github.com/home-assistant/core/pull/71432
+
+{% enddetails %}
+
+{% details "Z-Wave" %}
+
```

Comment:
Should we add that the Z-Wave server should be updated ?

I bumped core to beta and then I got the repair message about that.

Sorry haven't found where the merge was made for https://rc.home-assistant.io/latest-release-notes/